src/autoscaler/scalingengine/scalingengine_test.go

```diff
@@ -92,7 +92,7 @@ var _ = Describe("ScalingEngine", func() {
 					Status:       models.ScalingStatusSucceeded,
 					OldInstances: 2,
 					NewInstances: 3,
-					Reason:       "+1 instance(s) because memoryused > 222 for 100 seconds",
+					Reason:       "+1 instance(s) because memoryutil > 80 for 100 seconds",
```

Comment:
this will need change the model of trigger to carry the metric unit information.  I will add this.